packages/ove-service-auth/heimdall/__init__.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+import falcon
+
+from heimdall.middleware import AuthMiddleware, LockMiddleware
+from heimdall.managers import UserManager, AccessManager, ServiceProxyManager, LockManager
+from heimdall.proxy import ProxyAdapter
+from heimdall.routes import LoginResource, LockResource
+from heimdall.util import parse_logging_lvl, random_string
+
+
+def setup_app(
+        jwt_key: str = random_string(32), jwt_token_issuer: str = "heimdall", jwt_token_expiration_seconds: int = 3600,
+        access_enabled: bool = True, access_config: str = "config/access.json",
+        login_enabled: bool = True, login_config: str = "config/users.json", login_hash_passwords: bool = True,
+        proxy_enabled: bool = True, proxy_config: str = "config/services.json", proxy_forward: str = "",
```

Comment:
Calling this argument `proxy_enabled` is confusing, because the proxy acts as proxy regardless of whether it is in `global` or `service` proxy mode.

It would be better to call this `service_proxy_enabled` (to match the environment variable that sets its value), or replace with a non-Boolean `proxy_mode`.

Reply:
As we discussed offline, this is still a work in progress and this is an unstable release that could be tested with the other ove services